extensions/amp-story/1.0/amp-story-page.js

```diff
@@ -425,6 +445,38 @@ export class AmpStoryPage extends AMP.BaseElement {
     return Promise.all(mediaPromises);
   }
 
+  /**
+   * Finds embedded components in page and prepares them for their expanded view
+   * animation.
+   * @param {number} timeout
+   * @private
+   */
+  findAndPrepareEmbeddedComponents_(timeout) {
+    scopedQuerySelectorAll(this.element, EMBEDDED_COMPONENTS_SELECTORS)
+        .forEach(el => {
+          if (!el.classList.contains('i-amphtml-embedded-component')) {
```

Comment:
How could something that matched `EMBEDDED_COMPONENTS_SELECTORS` not be an `i-amphtml-embedded-component`?

Reply:
I think it'd be cool to have another way to know that they're resized, it's really not obvious here

Reply:
Done.